cs481-ekh/f21-welolo
# Task#47

This ticket just broke out our transaction endpoint into multiple other endpoints to help assist us in modularity.

Verification Steps:
1. Run './build.sh' in the root directory of the project
2. Navigate to both the Client and Server folders in two separate terminals
3. Run 'npm start' in both folders
4. Navigate to localhost:3000
5. Open the developer tool and navigate to the 'Networks' tab
5. Enter some value into all fields, but enter '+15005550000' into the Recipient # field
6. Click submit
7. Notice there are two different requests that are being made, 'send_payment' and 'send_sms'
8. Verify the response from both requests is '{ success: true }'
